engine/src/main/java/io/camunda/zeebe/engine/state/migration/DbMigrator.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.engine.state.migration;
+
+public interface DbMigrator {
```

Comment:
It looks like there's only one implementation, but if we expect multiple implementations, I would document the contract of this interface. Otherwise it's fine.

Reply:
There is just one implementation. I don't expect any other implementations in the future. The interface is because we have the final classes policy and the don't use Powermockito policy. So the interface is mainly so I can mock it in tests.